src/static/containers/Dashboard.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { connect, PromiseState } from 'react-refetch';
+import PropTypes from 'prop-types';
+import { Col, Row } from 'antd';
+import ZeroCard from './ZeroCard';
+
+import './Dashboard.css';
+
+const Dashboard = ({ usercardsFetch, refreshUserCards }) => {
+    if (usercardsFetch.pending) {
+        return <p className="text-center">Loading cards...</p>;
+    } else if (usercardsFetch.fulfilled) {
+        const cards = usercardsFetch.value;
+        if (cards.length) {
```

Comment:
You must check if cards is ok and maybe check the instance of it like:
cards && cards instanceof Array
Or something like that